[ZEPPELIN-998] Extend install.md -> Quick Start

AhyoungRyu opened the pull request:
### What is this PR for?
Most of other projects have **Quick Start** or **Getting Started** page for the beginner. Currently, Zeppelin also has [Zeppelin Install](https://zeppelin.apache.org/docs/0.6.0-SNAPSHOT/install/install.html) which is similar with those kind of instruction page. But it has only contents that explain just installation and configuration. So I updated this page to **Quick Start** so that it can include step by step guide for the beginners.

### What type of PR is it?
Improvement & Documentation 

### Todos
* [x] - Add each title link to the head of documentation 
* [x] - Add more information about Zeppelin installation
* [x] - Reorder contents 

### What is the Jira issue?
[ZEPPELIN-998](https://issues.apache.org/jira/browse/ZEPPELIN-998)

### How should this be tested?
See the attached screenshot images

### Screenshots (if appropriate)
<img width="809" alt="screen shot 2016-06-14 at 2 59 47 pm" src="https://cloud.githubusercontent.com/assets/10060731/16061227/d88abe56-3240-11e6-845c-e37a9975aceb.png">
<img width="796" alt="screen shot 2016-06-14 at 3 00 01 pm" src="https://cloud.githubusercontent.com/assets/10060731/16061229/daca8b06-3240-11e6-821b-7d118b7b3e09.png">
<img width="789" alt="screen shot 2016-06-14 at 3 00 15 pm" src="https://cloud.githubusercontent.com/assets/10060731/16061234/dd12072c-3240-11e6-9a0a-cf1e320fd879.png">
<img width="785" alt="screen shot 2016-06-14 at 3 00 27 pm" src="https://cloud.githubusercontent.com/assets/10060731/16061238/df7f904c-3240-11e6-83e0-73f3688c0815.png">
<img width="789" alt="screen shot 2016-06-14 at 3 00 38 pm" src="https://cloud.githubusercontent.com/assets/10060731/16061242/e1ad1402-3240-11e6-81ba-2e7125cec98e.png">

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

bzz commented on docs/install/install.md:
````diff
@@ -19,30 +19,173 @@ limitations under the License.
+```
+
+## What is the next?
+Congratulation to your successful Zeppelin installation! Here are two next steps you might need.
````
I belive it should be `Congratulation _ob_` as it's not an Apache Zeppelin installation that is being congratulated here, but the user.

AhyoungRyu replied:
@bzz Yeah you're right. I just updated it :)

bzz commented:
@AhyoungRyu thank you for great effort structuring the docs!

It looks awesome, just two things that I have noticed:
 1. can you please double check that project is referred as Apache Zeppelin everywhere (following branding guidelines)
 2. having `npm` installed is not a requirement to build project from sources

  `mvn package` should be enough and will take care of installing both, local node as well as NPM so I would suggest we remove mention of npm from 'building from source section'

AhyoungRyu commented:
@bzz Really appreciated to your review again :) 
 - I added `Apache` before the most of `Zeppelin`. 
 - I didn't noticed that "npm, downloaded by Maven during build phase" in [README.md#requirements](https://github.com/apache/zeppelin/blob/master/README.md#requirements). So I removed it as you said.

bzz commented:
@AhyoungRyu great job! Let me take another pass on it plz.

BTW is removal of `conf/shiro.ini` a part of the quickstart?

bzz commented on docs/install/install.md:
```diff
@@ -19,38 +19,113 @@ limitations under the License.
-Welcome to your first trial to explore Zeppelin!
+# Quick Start
+Welcome to your first trial to explore Apache Zeppelin! 
+This page will help you to get started with Apache Zeppelin. Here is the list of this page. 
```
This page will help you to get started and here is the list of topics covered

AhyoungRyu commented:
@bzz Oh my. That's my bad. I'll revert it now :)

bzz commented on docs/install/install.md:
```diff
@@ -19,38 +19,113 @@ limitations under the License.
+## Installation
 
-If you want to install Zeppelin with latest binary package, please visit [this page](http://zeppelin.apache.org/download.html).
+Apache Zeppelin officially tested on the below environment. 
```
Apache Zeppelin officially supports and is tested on next environments

bzz commented on docs/install/install.md:
```diff
@@ -19,38 +19,113 @@ limitations under the License.
+After unpacking, jump to [Starting Apache Zeppelin with Command Line](#starting-apache-zeppelin-with-command-line) section.
+
+### Building from Source
+If you want to build from the source, there are more requirements. 
```
If you want to build from the source, the software below needs to be installed on your system

bzz commented on docs/install/install.md:
```diff
@@ -19,38 +19,113 @@ limitations under the License.
+  </tr>
+</table>
+
+If you don't have the above requirements yet, please check [Before Build](https://github.com/apache/zeppelin/blob/master/README.md#before-build) section and follow step by step.
```
If you don't have it installed yet, please check... and follow step by step instructions from there.

bzz commented:
Thanks for prompt response! 
Your documentation update looks great to me, modulo few minor suggestions above.

Please feel free to use them at your will, otherwise I think it's ready to be merged.

AhyoungRyu commented:
@bzz Thanks for all of your suggestions again. I've just addressed them. Please check my last commit :)

bzz commented:
Looks great to me, thank you @AhyoungRyu 

Will merge to master, if there is no further discussion
